nixos/modules/services/games/minecraft-server.nix

```diff
@@ -4,6 +4,68 @@ with lib;
 
 let
   cfg = config.services.minecraft-server;
+  eulaFile = pkgs.writeText "eula.txt" ''
+    # eula.txt manged by NixOS Configuration
+    eula=${if cfg.eula then "true" else "false"}
+  '';
+  whitelistFile = pkgs.writeText "whitelist.json" (builtins.toJSON (lib.mapAttrsToList (n: v: { uuid = n; name = v; }) (if cfg.whitelist != null then cfg.whitelist else {}) ));
```

Comment:
If you're gonna use `{}` as the default value if the whitelist is `null` as a default, you may as well make `{}` the default value of the whitelist and not have this `if` here.